src/components/Composer/index.ios.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {ForwardedRef, useCallback, useEffect, useMemo, useRef} from 'react';
+import {StyleSheet, TextInput} from 'react-native';
+import RNTextInput from '@components/RNTextInput';
+import * as ComposerUtils from '@libs/ComposerUtils';
+import useTheme from '@styles/themes/useTheme';
+import useThemeStyles from '@styles/useThemeStyles';
+import {ComposerProps} from './types';
+
+function Composer(
+    {
+        shouldClear = false,
+        onClear = () => {},
+        isDisabled = false,
+        maxLines,
+        isComposerFullSize = false,
+        setIsFullComposerAvailable = () => {},
+        autoFocus = false,
+        isFullComposerAvailable = false,
+        style,
+        ...props
+    }: ComposerProps,
+    ref: ForwardedRef<TextInput>,
+) {
+    const textInput = useRef<TextInput>();
```

Comment:
```suggestion
    const textInput = useRef<TextInput>(null);
```

Let's bring back the `null`.

Reply:
Updated